src/main/resources/petriNets/engine-processes/filter.xml

```diff
@@ -396,10 +396,10 @@
         <id>frontend_create</id>
         <x>380</x>
         <y>260</y>
+        <label>Set filter metadata</label>
```

Comment:
Are the following tasks only system tasks? Do labels need translation?

Reply:
These are system tasks and are automated. They don't need translations

Reply:
Ok